app/helpers/application_helper.rb

```diff
@@ -157,6 +157,14 @@ def cache_key_heroku_slug(path)
     "#{path}-#{heroku_slug_commit}"
   end
 
+  def copyright_notice
+    start_year = ApplicationConfig["COMMUNITY_COPYRIGHT_START_YEAR"]
+    current_year = Time.current.year
+    return start_year if current_year == start_year
+
+    "#{start_year} - #{current_year}"
```

Comment:
it's improbable it will happen but if `COMMUNITY_COPYRIGHT_START_YEAR` is set to an empty string (some sites only display the latest year in the copyright notice?), this will render ` - 2020`

Reply:
We set the default to 2016, but I guess someone could set it as an empty string. It's an edge case but we could just add something like ```return current_year if start_year.strip! == nil'``` or check the length to cater for it. What do you think?